cmd/table.go

```diff
@@ -63,12 +79,12 @@ func unquotedPath(path ast.Ref) []string {
 	return ret
 }
 
-func createTable(args []string, params tableCommandParams) error {
+func createTable(args []string) (bytes.Buffer, error) {
```

Comment:
[nit] I think it would be slightly more idiomatic to return `io.Reader` here. Since it's an interface, you can then also use the much-beloved `return nil, err` instead

Reply:
Updated in all locations 👍